src/main/java/it/pagopa/transactions/utils/NodoOperations.java

```diff
@@ -39,6 +42,13 @@ public class NodoOperations {
     @Autowired
     NodoConfig nodoConfig;
 
+    @Value("${nodo.lightAllCCPCheck}")
+    private boolean lightAllCCPCheck;
+
+    void setLightAllCCPCheck(boolean lightAllCCPCheck) {
```

Comment:
Is this method needed?

Reply:
I need this method for test purpose, to test both ways of enabled and disabled flag

Reply:
Maybe `ReflectionTestUtils` could help you, like https://github.com/pagopa/pagopa-ecommerce-transactions-service/blob/3471c4da62fda6339f79ea4cedf53bd0b9f70a79/src/test/java/it/pagopa/transactions/configurations/NodoConfigTest.java#L22

Reply:
Or here can be used constructor injection in place of field injection if you want to avoid reflection usage in tests :)

Reply:
yes, to be preferred to reflection

Reply:
so, for example https://github.com/pagopa/pagopa-ecommerce-transactions-service/blob/3471c4da62fda6339f79ea4cedf53bd0b9f70a79/src/main/java/it/pagopa/transactions/commands/handlers/TransactionRequestAuthorizationHandler.java#L42